### DISCUSSION TITLE
ESM2 Pre-training Dataset Split description

### DISCUSSION THREAD

**Discussion by erm96** [facebookresearch/esm#437] [Q&A]
Hi, 

is there a description of the dataset split for ESM2 (UniRef Ids of the evaluation set) ? 
The paper only mentions 
> UniRef50, September 2021 version, is used for the training
> of ESM models. The training dataset was partitioned by
> randomly selecting 0.5% (≈ 250,000) sequences to form
> the validation set.

I could only find a description of UniRef Ids of the evaluation set for ESM1 on the main git page.

Really appreciate your awesome work!

**Comment by ijayden-lung** [comment]
I have the same questions. What's the ESM-2 validation set
